### PR TITLE
final doc updates for 7/2/12 EAP release

### PR DESCRIPTION
**Pull request by masthalter**

### CODE REVIEW

**Comment by cp16net** [comment]
+1

File: apidocs/src/resources/samples/db-credentials-20.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+Accept: application/json
+Content-Type: application/json
+Content-Length: 54
+
+{
+   "auth":
+   {
+      "RAX-KSKEY:apiKeyCredentials":
+      {
+         "username": "hub_cap",
```
Comment (ed-) [inline apidocs/src/resources/samples/db-credentials-20.json:13]: Sample or not, should this be dummied, or is it already?

File: apidocs/src/resources/samples/db-credentials-20.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+Host: identity.api.rackspacecloud.com
+Accept: application/xml
+Content-Type: application/xml
+Content-Length: 88
+
+<?xml version="1.0" encoding="UTF-8"?>
+<auth>   
+   <apiKeyCredentials     
+       xmlns="http://docs.rackspace.com/identity/api/ext/RAX-KSKEY/v1.0"     
+       username="hub_cap"
```
Comment (ed-) [inline apidocs/src/resources/samples/db-credentials-20.xml:12]: Same here re: dummying

File: apidocs/src/resources/samples/db-credentials-20.json (hunk 1 above)
Comment (masthalter) [inline apidocs/src/resources/samples/db-credentials-20.json:13]: Hi Ed,

I assume that you're posing this question to the DB Team and not to me
specifically. Is that correct, Ed?

Thanks,

Mike

On 7/3/12 3:12 PM, "Ed Cranford"
<reply+p-1096064-083f8c550de1e840972fbe1b32ce59c3016289d2-1015169@reply.git
hub.com> wrote:

Comment (ed-) [inline apidocs/src/resources/samples/db-credentials-20.json:13]: Right, I'm doing it here to be visible.

**Comment by imsplitbit** [comment]
yeah I think dummying it would be better than using hub_cap's nick

**Comment by ed-** [comment]
Using his nick isn't the problem since he's also the repo owner. It's giving away his api key that I was worried about, but as long as @hub-cap can confirm that the api keys here aren't valid, then

+1

File: apidocs/src/resources/samples/db-credentials-20.xml (hunk 2 above)
Comment (rnirmal) [inline apidocs/src/resources/samples/db-credentials-20.xml:12]: yeah that's just a dummy api key. But you are right we should probably change that .

Comment (masthalter) [inline apidocs/src/resources/samples/db-credentials-20.xml:12]: Hi all,

Here is what is used in the Beta version of the Cloud Identity Client
Developer Guide:

<?xml version="1.0" encoding="UTF-8"?>
<auth>
  <apiKeyCredentials
    xmlns="http://docs.rackspace.com/identity/api/ext/RAX-KSKEY/v1.0"
    username="jsmith"
    apiKey="aaaaa-bbbbb-ccccc-12345678"/>
  </auth>

I would be happy to change to this for the next version of the doc if you
all prefer.

Please advise,

Mike

On 7/5/12 11:23 AM, "Nirmal Ranganathan"
<reply+p-1106263-90f89f6d31b89094c3c0826baeae7fbf1114bcc6-1015169@reply.git
hub.com> wrote:

Comment (masthalter) [inline apidocs/src/resources/samples/db-credentials-20.xml:12]: doesnt bother me either way :)

Comment (ed-) [inline apidocs/src/resources/samples/db-credentials-20.xml:12]: Perfect

**Comment by rnirmal** [comment]
Merging this in as this has already been published. We can change the username later if needed.

**Comment by masthalter** [comment]
Thanks!

Mike

On 7/5/12 11:52 AM, "Nirmal Ranganathan"
<reply+i-5410581-84055642c21757953502b70be19f20129fbd2f11-1015169@reply.git
hub.com> wrote:

**Comment by imsplitbit** [comment]
I still think it's not appropriate for docs to have his username, they should have things like <your username> and <your apikey> no?

**Pull request merged**